### ISSUE TITLE
[BUG] Crash: VerifyErrorRejecting class com.android.quickstep.views.LauncherRecentsView

### ISSUE THREAD

**Issue by Goooler** [Deletescape-Media/Lawnchair#3167]
### Describe the bug

```
Process: app.lawnchair, PID: 22400
java.lang.VerifyError: Rejecting class com.android.quickstep.views.LauncherRecentsView that attempts to sub-type erroneous class 
com.android.quickstep.views.RecentsView (declaration of 'com.android.quickstep.views.LauncherRecentsView' appears in base.apk!classes2.dex)
    at java.lang.reflect.Constructor.newInstance0(Native Method)
    at java.lang.reflect.Constructor.newInstance(Constructor.java:343)
    at android.view.LayoutInflater.createView(LayoutInflater.java:647)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:790)
    at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
    at android.view.LayoutInflater.parseInclude(LayoutInflater.java:963)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:859)
    at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
    at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
    at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
    at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
    at com.android.launcher3.statemanager.StatefulActivity.inflateRootView(Unknown Source:5)
    at com.android.launcher3.Launcher.onCreate(Unknown Source:186)
    at com.android.launcher3.BaseQuickstepLauncher.onCreate(Unknown Source:0)
    at app.lawnchair.LawnchairLauncher.onCreate(Unknown Source:17)
    at android.app.Activity.performCreate(Activity.java:7327)
    at android.app.Activity.performCreate(Activity.java:7318)
    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3096)
    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3259)
    at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
    at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1950)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loop(Looper.java:214)
    at android.app.ActivityThread.main(ActivityThread.java:7073)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:964)
```

### Steps to reproduce

1. `git clone --recursive https://github.com/LawnchairLauncher/lawnchair.git`
2. `cd lawnchair`
3. `git checkout disable_cc_for_r8`
4. `./gradlew assembleLawnWithQuickstepRelease`
5. Install the apk from `./build/outputs/apk/lawnWithQuickstep/release/` on Android 8 or 9 devices.
6. Open the Lawnchair, and you'll see the crash.

### Expected behavior

No crash

### Device information

- Device: Samsung S8
- OS: Android 8 & 9
- App version: d1fc5a53509afebe8a43eca3f84a23976facda5a

**Comment by Goooler** [comment]
Seems happened after AGP updating from 7.2.2 to 7.3.0+

**Comment by Goooler** [comment]
Keep rules already added in 

https://github.com/LawnchairLauncher/lawnchair/blob/ea43f4b6f57687a03b509716a3e77f11bde5bd58/proguard.pro#L50-L52

**Comment by Goooler** [comment]
Filed at https://issuetracker.google.com/issues/262114759.

**Comment by Goooler** [comment]
Would like to keep this issue open to wait the replay of Google instead of reverting AGP version.

**Comment by Goooler** [comment]
Fixed by #3186.
